docs/csharp/language-reference/keywords/sizeof.md

```diff
@@ -25,9 +25,9 @@ int intSize = sizeof(int);
 ```  
   
 ## Remarks  
- Starting with version 2.0 of C#, applying `sizeof` to built-in types no longer requires that [unsafe](../../../csharp/language-reference/keywords/unsafe.md) mode be used.  
+ Starting with version 2.0 of C#, applying `sizeof` to simple or enum types no longer requires that [unsafe](../../../csharp/language-reference/keywords/unsafe.md) mode be used.  
```

Comment:
\[unsafe](unsafe.md) mode be used --> code be compiled in an \[unsafe](unsafe.md) context